rosidl_typesupport_c/cmake/rosidl_typesupport_c_generate_interfaces.cmake

```diff
@@ -12,6 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+if(NOT rosidl_generator_c_FOUND)
+  message(FATAL_ERROR
+    "Executing rosidl_typesupport_c extension point when rosidl_generator_c wasn't found."
+    "Make sure rosidl_generator_c is installed.")
```

Comment:
Since this condition is unlikely to happen and when it happens it potentially is not because it is not installed but not declared as a dependency I would rather drop the second sentence.

PS: this line also fails the linter atm

Same below for C++.

Reply:
Done, see https://github.com/ros2/rosidl_typesupport/pull/73/commits/5d01aa94fad084f5b9f2316a3fbd3e42f8e91c3c.